etc/docker/kubeflow/Dockerfile

```diff
@@ -27,17 +27,17 @@ ARG ELYRA_PACKAGE=elyra-"$ELYRA_VERSION"-py3-none-any.whl
 ARG ELYRA_EXTRAS=[kfp-tekton,kfp-examples]
 
 # Includes the readme as a token file for COPY that always exists, otherwise production builds fail when whl not present
-COPY $ELYRA_PACKAGE README.md ./
+COPY README.md ${ELYRA_PACKAGE}* ./
 
 # Install Elyra
 RUN if [ "$TAG" = "dev" ] ; then \
-    python -m pip install --quiet --no-cache-dir "$ELYRA_PACKAGE""$ELYRA_EXTRAS"; \
+    python -m pip install --quiet --no-cache-dir "${ELYRA_PACKAGE}${ELYRA_EXTRAS}" ; \
     else \
-    python -m pip install --quiet --no-cache-dir elyra"$ELYRA_EXTRAS"=="$TAG" ; fi
+    python -m pip install --quiet --no-cache-dir "elyra${ELYRA_EXTRAS}"=="$TAG" ; fi
 
 # Cleanup
 USER root
-RUN rm $ELYRA_PACKAGE README.md
+RUN rm README.md ${ELYRA_PACKAGE}*
```

Comment:
Per the previous comment, can we then remove the wildcard from this `rm`?

I see the following relative to this command?
```
Step 9/11 : RUN rm README.md ${ELYRA_PACKAGE}*
 ---> Running in 474529d1080a
rm: cannot remove 'elyra-3.10.1-py3-none-any.whl*': No such file or directory
The command '/bin/bash -c rm README.md ${ELYRA_PACKAGE}*' returned a non-zero code: 1
(base) [wildcard_copy]$ 
```
Using `-f` and removing the wildcard appears to work...
```suggestion
RUN rm -f README.md ${ELYRA_PACKAGE}
```
```
Step 9/11 : RUN rm -f README.md ${ELYRA_PACKAGE}
 ---> Running in 37002c6dd016
Removing intermediate container 37002c6dd016
```